features/steps/steps.coffee

```diff
@@ -0,0 +1,20 @@
+jsdiffConsole = require 'jsdiff-console'
+ObservableProcess = require 'observable-process'
+
+
+module.exports = ->
+
+  @Given /^a Cucumber spec with missing step implementation for$/, (string) ->
+
+
+  @When /^running Cucucumber\-JS with this package enabled$/, (done) ->
+    command = '../node_modules/.bin/cucumber-js --snippet-syntax ../index --no-colors'
+    @observer = new ObservableProcess command, cwd: 'example_app'
```

Comment:
Rather than actually having an example_app directory I'd rather the scenario creates a temp directory and the given step writes the file in the temp directory, and then you run the command in the temp directory.